authorization-server/src/modules/auth/strategies/jwt.strategy.ts

```diff
@@ -1,22 +1,34 @@
-import { ExtractJwt, Strategy } from 'passport-jwt';
+import { ExtractJwt, JwtFromRequestFunction, Strategy } from 'passport-jwt';
 import { PassportStrategy } from '@nestjs/passport';
 import { Injectable } from '@nestjs/common';
 import { ConfigService } from '@nestjs/config';
+import * as express from 'express';
 
 @Injectable()
 export class JwtStrategy extends PassportStrategy(Strategy, 'jwt') {
   constructor(config: ConfigService) {
+    const extractorFunctions: JwtFromRequestFunction[] = [];
+
+    const fromHeader = ExtractJwt.fromAuthHeaderAsBearerToken();
+
+    const fromCookie = (req: express.Request) =>
+      req &&
+      req.cookies &&
+      req.cookies[config.get<string>('AUTH_COOKIE_NAME_ACCESS_TOKEN')];
```

Comment:
Small change, but one check here can be removed I think.
```suggestion
      req?.cookies &&
      req.cookies[config.get<string>('AUTH_COOKIE_NAME_ACCESS_TOKEN')];
```

Reply:
@artursudnik I think the point of your comment above is that `req` could be `null`.
In this case, wouldn't `req?.cookie` evaluate to `undefined` and so the check would work?

I did a quick test in node REPL and it seems to work
```
❯ node
Welcome to Node.js v18.14.0.
Type ".help" for more information.
> const req = null;
undefined
> req?.cookies
undefined
> req
null
```

Reply:
oh, you are right. I did not notice added question mark. I am changing this